src/main/java/uk/ac/cam/cl/dtg/isaac/api/managers/EventBookingManager.java

```diff
@@ -1452,4 +1489,19 @@ private void removeUserFromEventGroup(final IsaacEventPageDTO event, final Regis
           user.getEmail(), event.getTitle()));
     }
   }
+
+  private static final String EXCEPTION_MESSAGE_TEMPLATE_DUPLICATE_BOOKING =
+      "Unable to book onto event (%s) as user (%s) is already booked on to it.";
+  private static final String EXCEPTION_MESSAGE_TEMPLATE_CANCELLED_EVENT =
+      "Unable to book user (%s) onto event (%s); the event is cancelled.";
+  private static final String EXCEPTION_MESSAGE_TEMPLATE_UNABLE_TO_SEND_EMAIL =
+      "Unable to send event email ({}) to user ({})";
+  private static final String EMAIL_TEMPLATE_ID_EVENT_BOOKING_CONFIRMED = "email-event-booking-confirmed";
+  private static final String EMAIL_TEMPLATE_ID_WAITING_LIST_ONLY_ADDITION =
+      "email-event-waiting-list-only-addition-notification";
+  private static final String EMAIL_TEMPLATE_ID_WAITING_LIST_ADDITION =
+      "email-event-waiting-list-addition-notification";
+  private static final String EMAIL_TEMPLATE_TOKEN_EVENT_URL = "eventURL";
+  private static final String AUTH_TOKEN_LINK = "https://%s/account?authToken=%s";
+  private static final String EVENT_STAGE_STUDENT = "student";
```

Comment:
am I misreading this, or is this a slightly odd place to put a bunch of constants?